cdk/lib/gatehouse.ts

```diff
@@ -213,5 +238,93 @@ export class Gatehouse extends GuStack {
 			domainName: props.domainName,
 			resourceRecord: app.loadBalancer.loadBalancerDnsName,
 		});
+
+		const vpc = GuVpc.fromIdParameter(this, 'IdentityVPC');
+		const rdsSecurityGroupRules = new SecurityGroup(
+			this,
+			'RDSSecurityGroupRules',
+			{
+				vpc: vpc,
+				allowAllOutbound: false,
+			},
+		);
+
+		const rdsSecurityGroupClients = new SecurityGroup(
+			this,
+			'RDSSecurityGroupClients',
+			{
+				vpc: vpc,
+				allowAllOutbound: false,
+				description: 'Allow access to Gatehouse DB from Clients',
+			},
+		);
+
+		rdsSecurityGroupRules.addIngressRule(
+			rdsSecurityGroupClients,
+			Port.tcp(databasePort),
+		);
+
+		const cluster = new DatabaseCluster(this, 'GatehouseDb', {
+			engine: DatabaseClusterEngine.auroraPostgres({
+				version: AuroraPostgresEngineVersion.VER_15_10,
```

Comment:
Any specific reason we're using `15.10`?

Isn't the latest version `VER_17_2`?

Reply:
Should we consider using one of the LTS versions amazon recommends? https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.Updates.LTS.html

Especially as some of the features of LTS versions match our use case, namely "You can't afford downtime on your Aurora PostgreSQL application for upgrades outside of rare occurrences for critical patches.
"

Reply:
Ah yea, good catch. I wanted to test the upgrade path for a major version bump at one point to see what kind of downtime (if any) there would be. Kind of hard to do this without an application to test with though. I'll bump it up to the latest stable for now!

Reply:
Postgres 17 looks to be in preview atm, so will bump to 16.6 for now! https://docs.aws.amazon.com/AmazonRDS/latest/AuroraPostgreSQLReleaseNotes/AuroraPostgreSQL.Updates.html#aurorapostgresql-versions-version17

Reply:
Database spent like 20 minutes offline and then failed with "Database cluster is in a state that cannot be upgraded: Postgres cluster is in a state where pg_upgrade can not be completed successfully."

Well thats not great.

Reply:
Increased the maximum capacity of the cluster and the upgraded worked this time. Seems like the upgrade was just running out of memory.

Worth noting that according to AWS the cluster was offline for 13 minutes for the upgrade.